_posts/2019-11-07-open-source-camp-on-foreman-2019.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: Open Source Camp on Foreman 2019 Review
+date: 2019-11-7 17:00:00
+author:
+ name: Alexander Stoll
+ uri: Crited
+tags:
+- foreman
+- event
+- meetup
+- community
+guest_post: true
+guest_url: https://www.netways.de/blog/2019/11/07/open-source-camp-on-foreman/
+excerpt: |
+ A quick recap of the Open Source Camp on Foreman following the OSMC.
+---
+
+## Open Source Camp on Foreman
+
+<img src="https://www.netways.de/wp-content/uploads/2019/11/201911_OSCAMP_Foreman_Blog_02.png" style="max-width:100%;float: right;align:center">
```

Comment:
This image seems to look strange and breaks the flow, unlike how it is displayed on the netways blog:
![image](https://user-images.githubusercontent.com/1540531/68864056-25431c80-06f9-11ea-9a64-cef0e9bd9f49.png)
perhaps we should skip it here?

Reply:
I changed it to look like this
![Preview](https://user-images.githubusercontent.com/45196568/69045063-63408900-09f6-11ea-986d-5beabab16e17.png)

I have nothing against removing it, though. Your call. :)

Reply:
That still looks quite large in here and not as good as it does on the netways blog, so i think it would be good to remove it. I also now notice that the "open source camp on foreman" header above it is basically a duplicate of the title, so that can also be removed.

Reply:
Done and done. Now it looks like this:

![Update](https://user-images.githubusercontent.com/45196568/69215541-4cbb3e80-0b6a-11ea-8832-be6ca78d1443.png)